netbout-web/src/test/java/com/netbout/rest/bout/TkBoutITCase.java

```diff
@@ -98,4 +100,38 @@ public void showsBoutNotFound() throws Exception {
             .assertStatus(HttpURLConnection.HTTP_MOVED_PERM);
     }
 
+    /**
+     * TkBout can kick missing user and add flash message with level severe.
```

Comment:
@original-brownbear can we change word `add` here to "display" or "respond with"?